cgminer.c

```diff
@@ -1945,6 +1942,9 @@ static struct opt_table opt_config_table[] = {
 		     opt_hidden
 #endif
 		    ),
+	OPT_WITH_ARG("--htr-address",
+		     opt_set_charp, NULL, &opt_htr_address,
+		     "Set bitcoin target address when solo mining to bitcoind (mandatory)"),
```

Comment:
Can you please update the comments? If should be "Set hathor target address when solo mining (mandatory)".

Reply:
Fixed it